webui/src/pages/repositories/wizard/sparkQuickstartWizard.jsx

```diff
@@ -0,0 +1,85 @@
+import Wizard from "./wizard";
+import React, {useState} from "react";
+import {useAPI} from "../../../lib/hooks/api";
+import {config} from "../../../lib/api";
+import {Loading} from "../../../lib/components/controls";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import {Spinner} from "react-bootstrap";
+import Alert from "react-bootstrap/Alert";
+import RepositoryCreateForm from "../../../lib/components/repositoryCreateForm";
+
+const RepositoryCreationStep = ({repoCreationError, createRepo, onCancel, onComplete}) => {
```

Comment:
This step will probably be relevant to any template we will implement. Consider exporting it to another file, but if you prefer doing it once relevant I totally get it :)

Reply:
interesting